.github/workflows/claude.yml

```diff
@@ -0,0 +1,64 @@
+name: Claude Code
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
Can you follow the same style I have in the other workflows? instead of `[created]` it should be `- created`